auth/auth.go

```diff
@@ -97,6 +98,10 @@ func Register(r *http.Request) (bool, error) {
 	u.Hash = string(h)
 	u.ApiKey = GenerateSecureKey()
 	err = models.PutUser(&u)
+	if err != nil {
```

Comment:
Let's leave out changes to this file, since the only addition is an error check. We can keep things scoped to just the changes being made.